tools/scripts/update-bumpenvs-yaml.py

```diff
@@ -87,6 +87,8 @@ def get_all_builds(commit: str) -> Dict[str, Build]:
     for build_meta in req.json():
         if build_meta["vcs_revision"] == commit:
```

Comment:
Can we put the check (and print statement) here instead?  Like before parsing the build_meta?  Seems convoluted to have one check, then parse the build (where we print a message), then to have another check after parsing.

Reply:
done